modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -393,3 +393,54 @@ spacewalk-services restart
 ====
 Increasing this value will cause {productname} Server to take longer to check if a minion is unreachable or unresponsive, causing the {productname} Server to be slower or less responsive overall.
 ====
+
+
+== Salt 3000 to Salt Bundle Migration
+
+
+=== Switch {sles} 12 or {centos} 7 minions (Salt 3000 EOL) to Salt Bundle.
+
+.Procedure: Switching with [literal]``util.mgr_switch_to_venv_minion`` state to [package]``venv-salt-minion``
+
+. Apply [literal]``util.mgr_switch_to_venv_minion`` with no pillar specified first.
+  This will result in the switch to [package]``venv-salt-minion`` with copying configuration files etc.
+  It will not clean up the original [package]``salt-minion`` configurations and its packages.
++
+----
+salt <minion_id> state.apply util.mgr_switch_to_venv_minion
+----
+. Apply [literal]``util.mgr_switch_to_venv_minion`` with [literal]``mgr_purge_non_venv_salt`` set to [literal]``True`` to remove [package]``salt-minion`` and with [literal]``mgr_purge_non_venv_salt_files`` set to [literal]``True`` to remove all the files related to [package]``salt-minion``.
+This second step ensures the first step was processed, and then removes the old configuration files and the now obsolete [package]``salt-minion`` package.
++
+----
+salt <minion_id> state.apply util.mgr_switch_to_venv_minion pillar='{"mgr_purge_non_venv_salt_files": True, "mgr_purge_non_venv_salt": True}'
+----
+
+=== Switch {sles} 12 or {centos} 7 SSH minions (Salt 3000 EOL) to Salt Bundle.
+
+.Procedure: Enabling the Salt Bundle with Salt SSH support
+
+. Make sure that the bootstrap repository is created
+The bootstrap repository must be created before bootstrapping the client with {webui}. Salt SSH is using the Salt Bundle taken from the bootstrap repository based on the detected target operating system.
```

Comment:
The transition for Salt SSH from `py27-compat-salt` / Salt 3000 to the Salt Bundle is not only about bootstrapping, it's for all Salt SSH uses that target SLE12/RHEL7. I wonder how this part will be understood, I imagine readers will skip over the paragraph after reading the first sentence.

Maybe we can leave the bootstrapping with webui part out and just say something like "create the bootstrap repo with the venv-salt-minion package for SLE12/RHEL7, it's required for Salt SSH with the bundle" (with better wording)

Reply:
Maybe it's clearer now!